src/tiled/replaceobjectstile.h

```diff
@@ -0,0 +1,42 @@
+#ifndef REPLACEOBJECTSTILE_H
+#define REPLACEOBJECTSTILE_H
+
+#include "undocommands.h"
+#include "mapobject.h"
+
+#include <QList>
+#include <QUndoCommand>
+
+namespace Tiled {
+
+class Tile;
+
+namespace Internal {
+
+class MapDocument;
+
+class ReplaceObjectsTile : public QUndoCommand
```

Comment:
I think the class should be called `ChangeMapObjectsTile` and it can be moved into the existing `changemapobject.h/cpp` files.